index.html

```diff
@@ -25,7 +25,15 @@
     <main>
       <!--1st section - Hero -->
       <section class="section">
-        <div class="container"></div>
+        <div class="container">
+          <h1 class="hero-title">This website is awesome
```

Comment:
I think I would keep "h1" tag in one line because it doesn't seem that long - do as you believe is better :)